draft-kazuho-httpbis-priority.md

```diff
@@ -98,6 +80,117 @@ The terms sh-token and sh-boolean are imported from
 Example HTTP requests and responses use the HTTP/2-style formatting from
 {{?RFC7540}}.
 
+# Motivation for Replacing HTTP/2 Priorities {#motivation}
+
+An important feature of any implementation of a protocol that provides
+multiplexing is the ability to prioritize the sending of information. This was
+an important realization in the design of HTTP/2. Prioritization is a
+difficult problem, so it will always be suboptimal, particularly if one endpoint
+operates in ignorance of the needs of its peer.
+
+HTTP/2 introduced a complex prioritization signaling scheme that used a
+combination of dependencies and weights, formed into an unbalanced tree. This
+scheme suffers from poor deployment and interoperability.
+
+The rich flexibility permitted in tree building is rarely excersised by clients;
+they generally either chosen a model optimized for a web use case or do nothing
```

Comment:
Maybe there is an error in tense? Something like "they generally either have chosen .. ore have done nothing" might flow better.

Reply:
On a reread, I noticed an opportunity to make this paragraph stonger by stealing text from the next one. PTAL at https://github.com/kazuho/draft-kazuho-httpbis-priority/pull/61/commits/7cfe822c09cfd9457d0acc95a1696d58a7cc1628